pkg/validation/internal/good_practices.go

```diff
@@ -75,3 +83,51 @@ func validateResourceRequests(csv *operatorsv1alpha1.ClusterServiceVersion) (err
 	}
 	return errs, warns
 }
+
+// validateHubChannels will check the channels. The motivation for the following check is to ensure that operators
+// authors knows if their operator bundles are or not respecting the Naming Convention Rules.
+// However, the operator authors still able to choose the names as please them.
+func validateHubChannels(channels []string) error {
+	const candidate = "candidate"
+	const stable = "stable"
+	const fast = "fast"
+
+	channels = getUniqueValues(channels)
+	var channelsNotFollowingConventional []string
+	for _, channel := range channels {
+		if !strings.HasPrefix(channel, candidate) &&
+			!strings.HasPrefix(channel, stable) &&
+			!strings.HasPrefix(channel, fast) &&
+			channel != "" {
+			channelsNotFollowingConventional = append(channelsNotFollowingConventional, channel)
+		}
+
+	}
+
+	if len(channelsNotFollowingConventional) > 0 {
+		return fmt.Errorf("channel(s) %+q are not following the recommended naming convention: "+
```

Comment:
Should this point to https://olm.operatorframework.io/docs/best-practices/channel-naming/#recommended-channel-naming instead?

Reply:
The whole doc is about this topic.
So, I would prefer to link the doc instead of a topic from the doc since is more unlike we change the doc link then remove/rename some section on it.